package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gulp-ejs",
-  "version": "3.1.1",
+  "version": "3.1.2",
```

Comment:
Please remove this change, I'll do the version increment in a separate pull request.